src/plugins/audiobridge-plugin.js

```diff
@@ -787,6 +798,50 @@ class AudioBridgeHandle extends Handle {
     throw (error);
   }
 
+  /**
+   * 
+   * @param {object} params
+   * @param {number|string} params.room - The involved room
+   * @param {string} [params.secret] - The optional secret needed to manage the room
+   * @returns {} - Returns event as success or error
```

Comment:
This should return something like

`Promise<module:audiobridge-plugin~AUDIOBRIDGE_EVENT_MUTE_ROOM_RESPONSE>`

with a simple definition of `AUDIOBRIDGE_EVENT_MUTE_ROOM_RESPONSE` as
```
/**
 * The response event for audiobridge mute request.
 *
 * @typedef {object} AUDIOBRIDGE_EVENT_MUTE_ROOM_RESPONSE
 * @property {number|string} room - The involved room
 */
```